common/src/main/java/com/railwayteam/railways/ModSetup.java

```diff
@@ -52,5 +52,7 @@ public static void register() {
     BiomesOPlentyTrackCompat.register();
     DreamsAndDesiresTrackCompat.register();
     QuarkTrackCompat.register();
+    GarnishedTrackCompat.register();
+    HibernalHerbsTrackCompat.register();
```

Comment:
Missing imports for these

Reply:
> Missing imports for these

Are they not covered by this import?
![image](https://github.com/user-attachments/assets/a1250fec-b6ed-4953-9b1e-ce014e74210a)